<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>191</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0">
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>Start Timer</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="resetBtn">
       <property name="text">
        <string>Stop Timer</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="timerTxt">
       <property name="text">
        <string>00:00</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QTimeEdit" name="setTimer">
       <property name="currentSection">
        <enum>QDateTimeEdit::Section::MinuteSection</enum>
       </property>
       <property name="displayFormat">
        <string>mm:ss</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
